src/components/SlopedBackground/index.js

```diff
@@ -9,23 +10,46 @@ const SlopedSection = ({
   children,
   slopedBackgroundColor,
   slopedBackgroundImage,
+  dotBackground,
   ...props
-}) => (
-  <BackgroundWithNavigator
-    css={css`
-      ${background}
-      &::before {
-        ${slopedBackgroundImage &&
-          `background-image: ${slopedBackgroundImage};`}
-        ${slopedBackgroundColor &&
-          `background-color: ${slopedBackgroundColor};`}
-      }
-    `}
-    {...props}
-  >
-    {children}
-  </BackgroundWithNavigator>
-)
+}) => {
+  const theme = useTheme()
+  return (
+    <BackgroundWithNavigator
+      css={css`
```

Comment:
this `useTheme` is not necessary. The `css` props can be a function with theme as argument.

Reply:
fixed.